preact.config.js

```diff
@@ -12,6 +12,12 @@ const fsUtil = require('./fs-util');
  * @param {WebpackConfigHelpers} helpers - object with useful helpers when working with config.
  **/
 export default function(config, env, helpers) {
+
+	const critters = helpers.getPluginsByName(config, 'Critters')[0];
+	if (critters) {
+		critters.plugin.options.mergeStylesheets = false;
+	}
```

Comment:
```suggestion
	const critters = helpers.getPluginsByName(config, 'Critters')[0];
	if (critters) {
		critters.plugin.options.mergeStylesheets = false;
		critters.plugin.options.compress = false;
	}
```

This should fix it.

`mergeStylesheets` will ensure you don't end up with 1 big style tag

`compress` will ensure unused styles aren't removed. As you're not prerendering, Critters sees everything as "unused" and stripped them out. 

It does make the output a bit messy, but you can run a formatter over the template if necessary.